### DISCUSSION TITLE
SEO Related

### DISCUSSION THREAD

**Discussion by mazipan** [kawalcovid19/wargabantuwarga.com#147] [Ideas]
This discussion is to grouping all about SEO ideas and suggestions.

Some of pending issues:

- [x] https://github.com/kawalcovid19/wargabantuwarga.com/issues/24
- [x] https://github.com/kawalcovid19/wargabantuwarga.com/issues/44
- [x] https://github.com/kawalcovid19/wargabantuwarga.com/issues/115
- [x] https://github.com/kawalcovid19/wargabantuwarga.com/issues/158
- [x] #233 
- [x] https://github.com/kawalcovid19/wargabantuwarga.com/issues/234

**Comment by mazipan** [comment]
Need help to create a Google Search Console account, verify using HTML file, put on the `public` dir @zainfathoni

**Reply by zainfathoni** [reply]
Done, thanks for the suggestion, Mas @mazipan 🙏

**Comment by mazipan** [comment]
Already add sitemap on #220, but seems we need to create the robots.txt more dynamic.

Currently still using hardcoded value.
I will work on the other PR for this.

**Comment by mazipan** [comment]
#158 should be become next priority 0.

We already got a warning message on the Google Search Console.

**Reply by zainfathoni** [reply]
Shall we make an `epic` issue for this discussion, Mas @mazipan? So it's easier to track using GitHub's new [task list](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists) feature. 😁

**Reply by mazipan** [reply]
I just think to use a discussion, since my random idea maybe bothering all people watching the issues 😂

But ya, need to move to epic issue I think

**Reply by zainfathoni** [reply]
Definitely not, Mas @mazipan. Your idea has always been on the back of my mind as well. I just had no time to pay attention to these SEO-related issues, let alone creating and maintaining the issues. 😅

Therefore, I'm very grateful that you're taking these SEO-related matters into your own hands. 😁 🙏

**Reply by zainfathoni** [reply]
Epic created at #236 ✅

**Comment by mazipan** [comment]
![Screenshot_20210721-133138_Chrome](https://user-images.githubusercontent.com/7221389/126442156-42f22959-b1bc-4726-908f-10bca7318bf0.jpg)

For you information, we have 4 Sitemap with max size for each Sitemap is 1000.

So your Next.js App now produce 3.1K+ static pages (as you can see on the screenshot)

And it can be done around 6-7 minute build on Netlify (of course with additional Lighthouse score checker)

🙌

**Comment by mazipan** [comment]
A very helpful insight from @ekiriandra-seo

https://docs.google.com/document/d/1im7CF8XaF4yYQ_jJjjW7dOKZz8tJFvidtW7bOlqt1Y8/edit?usp=sharing

We got a lot of suggestions

**Reply by zainfathoni** [reply]
Thanks, @ekiriandra-seo for your suggestions!
